welcome/index.adoc

```diff
@@ -62,11 +68,11 @@ user-provisioned infrastructure on
 xref:../installing/installing_aws/installing-restricted-networks-aws.adoc#installing-restricted-networks-aws[AWS],
 GCP,
```

Comment:
Can "GCP" here link to https://deploy-preview-33427--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_gcp/installing-restricted-networks-gcp-installer-provisioned.html?